sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/SingleLifeEvent.kt

```diff
@@ -0,0 +1,8 @@
+package com.badoo.reaktive.samplemppmodule
+
+class SingleLifeEvent<out T : Any>(value: T) {
```

Comment:
Don't like it, it makes our state not immutable. Also you can face Kotlin's memory model restriction issues. See suggested implementation below.

Reply:
Well it is Google's approach in their MVVM. Will remove.